src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -252,6 +255,15 @@ public GitHubSCMNavigator(String apiUri, String repoOwner, String scanCredential
         }
     }
 
+    @CheckForNull
+    @Restricted(NoExternalUse.class)
+    public StandardCredentials getCredentials(@CheckForNull Item context, boolean forceRefresh) {
+        if (credentials == null || forceRefresh) {
```

Comment:
Hiding the method that currently not need to be used out from `GitHubSCMNavigator`
```suggestion
    private StandardCredentials getCredentials(@CheckForNull Item context, boolean forceRefresh) {
```